account_financial_report/i18n/es.po

```diff
@@ -6,46 +6,46 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Odoo Server 12.0+e\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2022-09-21 15:05+0000\n"
-"PO-Revision-Date: 2024-06-14 14:42+0000\n"
+"POT-Creation-Date: 2025-06-20 23:20+0000\n"
+"PO-Revision-Date: 2025-06-21 01:20+0200\n"
 "Last-Translator: Víctor Martínez <victor.martinez@tecnativa.com>\n"
 "Language-Team: \n"
 "Language: es\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=n != 1;\n"
-"X-Generator: Weblate 4.17\n"
+"X-Generator: Poedit 3.4.2\n"
 
 #. module: account_financial_report
 #: model_terms:ir.ui.view,arch_db:account_financial_report.report_aged_partner_balance_lines_header
 msgid "&gt; 120 d."
-msgstr "&gt; 120 d."
+msgstr ""
```

Comment:
These changes shouldn't be done in this PR. I think you have your POEdit to remove the translations equal to the source, but it's creating a big diff and marking the language as not translated, where it was.

Reply:
It's an option in pre-commit that was later deactivated, as it's not convenient. It saves bytes, but on the other hand, it provokes a lot of diffs and translation platform thinks that the language is not translated. In 15.0, it has never been activated.